backend/habitat/viewsets.py

```diff
@@ -531,32 +546,23 @@ def zones_iucn(request):
 @api_view()
 def habitat_statistics(request):
     network   = request.query_params.get('network')
+    network   = network if network else None
```

Comment:
I thought that it was weird that I had to do this, but `nil` query parameters in the frontend become empty strings in the backend, rather than the `None` I need. This is just here to fix that (I think it's the neatest solution?)